.travis.yml

```diff
@@ -11,18 +11,17 @@ notifications:
       - travis-build@platform-notifications.robomaker.aws.a2z.com
 env:
   global:
-    - UPSTREAM_WORKSPACE=file
-    - ROSINSTALL_FILENAME=.rosinstall.master
+    - GITHUB_ACCOUNT=aws-robotics
   matrix:
-    - ROS_DISTRO="kinetic" ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
-    - ROS_DISTRO="kinetic" ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
-    - ROS_DISTRO="melodic" ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
-    - ROS_DISTRO="melodic" ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
+    - ROS_DISTRO=kinetic ROS_VERSION=1
+    - ROS_DISTRO=melodic ROS_VERSION=1
+    - ROS_DISTRO=bouncy  ROS_VERSION=2
+    - ROS_DISTRO=crystal ROS_VERSION=2
 matrix:
   allow_failures:
-    - env: ROS_DISTRO="melodic" ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
-    - env: ROS_DISTRO="melodic" ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
+    - env: ROS_DISTRO=bouncy  ROS_VERSION=2
+    - env: ROS_DISTRO=crystal ROS_VERSION=2
 install:
-  - git clone https://github.com/ros-industrial/industrial_ci.git .ros_ci
+  - git clone https://github.com/aws-robotics/travis-scripts.git .ros_ci
```

Comment:
I think we should start moving `release-v*` branches to use their own branch of `travis-scripts`, different from the `master` branch.